content/docs/README.md

```diff
@@ -1,23 +1,19 @@
 ---
 title: cert-manager
-description: cert-manager documentation homepage
+description: |
+    cert-manager creates signed TLS / SSL certificates for workloads in your Kubernetes or OpenShift cluster
+    and renews the certificates before they expire.
 ---
 
-cert-manager adds certificates and certificate issuers as resource types in
-Kubernetes clusters, and simplifies the process of obtaining, renewing and
-using those certificates.
+cert-manager creates signed TLS / SSL certificates for workloads in your Kubernetes or OpenShift cluster
+and renews the certificates before they expire.
+Typically, the private key and signed certificate are stored in Kubernetes secrets
+and used by applications or ingress controllers.
 
-It can issue certificates from a variety of supported sources, including
-[Let's Encrypt](https://letsencrypt.org), [HashiCorp Vault](https://www.vaultproject.io),
-and [Venafi](https://www.venafi.com/) as well as private PKI.
+cert-manager can obtain the signed certificates from a variety of certificate authorities, including:
+[Let's Encrypt](configuration/acme/), [HashiCorp Vault](configuration/vault/),
+[Venafi](configuration/venafi/) and [private PKI](configuration/ca/).
```

Comment:
When I previewed these links locally, they always ended up with a double forward slash between `docs//configuration`, no matter how I wrote the link. E.g.

http://localhost:8888/docs//configuration/acme/

Any ideas why nextjs is doing this?

Reply:
https://github.com/cert-manager/website/#task-linking-to-other-documentation-pages

Does that help at all or is it something different to that?

Reply:
does `./configuration/acme/` work?

Reply:
I have tried both suggestions, but neither of them seem to work.
I've pushed a version where I link to the `.md` files, but this still has the `docs//configuration` problem and additionally, the resulting URl for the Let's Encrypt link becomes `http://localhost:8888/docs//configuration/acme/README/`

Reply:
@PatrickHeneise Do you have any idea what is causing the URLs to do this? :point_up:

Reply:
The style that I would expect to work would be `[Venafi](./configuration/venafi.md)` - everything linking to a markdown file with a relative link

Reply:
It's not redundant in this scenario AIUI

It's possible that there's some bug in the main README which breaks links to other pages from there

Reply:
Also locally, you can `Cmd+Click` on the link. If the file doesn't open, something is wrong.

Reply:
Thanks @PatrickHeneise , but is the case in this branch, and yet the URLs are wrong
 (double forward slashes and in the case of the acme link, a trailing README segment) 
See:
 * https://github.com/cert-manager/website/blob/baf88a31deb4c4dbd188e081d8e4fca65f47bf3b/content/docs/README.md
![image](https://user-images.githubusercontent.com/978965/189107499-e620ad71-83b8-4b0b-8a63-04c61f570076.png)



* https://deploy-preview-1071--cert-manager-website.netlify.app/docs/
![Screen Shot 2022-09-08 at 12 07 41](https://user-images.githubusercontent.com/978965/189107284-2c6ed781-e77c-44ca-b0d3-f08ddbc101a9.png)


![image](https://user-images.githubusercontent.com/978965/189107631-f1738f20-b5c7-4be4-831a-4c424c978637.png)

Reply:
@PatrickHeneise I suspect there's a bug in https://github.com/zentered/next-product-docs/blob/feat/cert-manager-adjustments/src/lib/remark-plugins/links/index.js , but I don't know how to make the nextjs / react / npm use a local copy of that package so that I can try and fix it.

Reply:
Hm. Could you prefix with `./`?

I'll probably work on the lib next week and can fix that (and write docs on how to load it locally). In the meantime, please add your case here: https://github.com/zentered/next-product-docs/blob/feat/cert-manager-adjustments/test/remark-plugins/links.test.js

Those are all the various cases we collected over time and they produce the correct links.

Reply:
We've started on the next release for next-product-docs :)

In the meantime, you can 
1. download the repo: `https://github.com/zentered/next-product-docs`
2. switch to the `feat/cert-manager-adjustments` branch
3. `pnpm install && pnpm build`
4. run `pnpm link .`
5. In the website repo, run `pnpm link @zentered/next-product-docs`

To boot up the development environment, run `pnpm watch` in the product-docs repo, that'll continuously build all the changes and make them available through the link.

Reply:
There's a new and simplified version of the internal link resolver you can find here: https://github.com/zentered/next-product-docs/blob/main/src/lib/remark-plugins/links/index.js

Test cases: https://github.com/zentered/next-product-docs/blob/main/test/remark-plugins/links.test.js

Reply:
see https://github.com/cert-manager/website/pull/1427